.clang-format

```diff
@@ -20,6 +20,11 @@ AllowShortFunctionsOnASingleLine: Inline
 AlignConsecutiveDeclarations: false
 AlignConsecutiveAssignments: false
 AlignConsecutiveMacros: Consecutive
+
+BinPackArguments: false
+AllowAllParametersOfDeclarationOnNextLine: false
+AllowAllArgumentsOnNextLine: false
```

Comment:
I think, I also fiddled around with these parameters in my previous commits, because the arguments list inconsistencies bothered me. But didn't want to change too much about the format rules back then.
tl;dr: This is fine.